xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/dropdown-menu.html

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="livedata-dropdown btn-group">
+
+    <a class="btn btn-default dropdown-toggle" title="More Actions" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true" role="button">
+      <span class="fa fa-ellipsis-v"></span>
+    </a>
+
+    <ul class="dropdown-menu">
+
+      <li class="dropdown-header">Change Layout</li>
```

Comment:
This needs to be translated. Same for "More Actions" above. The easiest is to use Velocity:

    $escapetool.xml($services.localization.render('someKey'))

and add ``evaluate=true`` as query string when loading this with RequrieJS.

Reply:
It's very strange I can't parse the velocity code, even with the `BASE_PATH` being `/xwiki/bin/skin/resources/uicomponents/livedata/`. I have currently no idea why it' not working, so I'm going to leave it like that, as it might probably be a configuration problem in the server, and so that you could have a look at my code to see if there is anything that seems wrong.

Reply:
Turns out translation is only working with webjars. (which is not very convenient)
As there are more important things to take care of atm, the issue is reported.